src/screens/Home/index.tsx

```diff
@@ -38,25 +40,25 @@ class Home extends Component {
       (note, i) =>
         i !== 0
           ? <LinkContainer
-              key={note.noteId}
-              to={`/notes/${note.noteId}`}
-            >
-              <ListGroupItem header={note.content.trim().split("\n")[0]}>
-                {"Created: " + new Date(note.createdAt).toLocaleString()}
-              </ListGroupItem>
-            </LinkContainer>
+            key={note.noteId}
+            to={`/notes/${note.noteId}`}
+          >
+            <ListGroupItem header={note.content.trim().split("\n")[0]}>
+              {"Created: " + new Date(note.createdAt).toLocaleString()}
+            </ListGroupItem>
+          </LinkContainer>
```

Comment:
The indent looks wrong

Reply:
Because of the IDE  😄

Reply:
Can you update your IDE and correct the file?